tuplex/historyserver/thserver/rest.py

```diff
@@ -254,28 +244,71 @@ def update_operator():
         inc_dict['detailed_ecounts.' + key] = val
 
     # upsert will create inc fields
-    mongo.db.operators.update_one({'jobid': ObjectId(jobid), 'id' : opid},
+    mongo.db.operators.update_one({'jobid': jobid, 'id' : opid},
                                   {'$inc': inc_dict},
                                   upsert=True)
 
     # fetch operator info
     status = mongo.db.operators.find_one({'jobid': ObjectId(jobid), 'id' : opid},
-                                    {'_id': 0,
-                                     'ncount': 1,
-                                     'ecount': 1})
+                                         {'_id': 0,
+                                          'ncount': 1,
+                                          'ecount': 1})
 
     assert status
     # query full and sent socketio update
     # send status update to all socketio clients
     status.update({'jobid' : jobid, 'opid' : opid})
     socketio.emit('operator_status', status)
+    print("ok")
 
     return jsonify({'status': 'ok'})
 
 
+def get_job(jobid):
```

Comment:
can we add documentation on what this function does? I.e. docstrings

Reply:
Yes. Docstrings have been added.